### ISSUE TITLE
Add information about GitHub/GitLab token changes after cluster creation

### ISSUE THREAD

**Issue by fharper** [konstructio/kubefirst-docs#625]
### What type of suggestions?

Missing Information

### What page is concerned?

FAQ

### What version of the docs?

2.3+

### What is the issue?

We need to write about what is happening if you need to recreate or change the GitHub or GitLab token after you created a Kubernetes cluster with kubefirst. It's something that can surely happen for security reason, or with the new GitHub token (don't know if GitLab has the same mechanesism) that expired after a set amount of time.

tl;dr

The user need to update the token for Atlantis in Vault, and everything should continue to work together.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fharper** [comment]
duplicate of #698
